ui/apps/platform/src/Containers/Policies/Wizard/Step3/PolicyGroupCard.tsx

```diff
@@ -68,7 +68,9 @@ function PolicyGroupCard({
         ]);
     }
 
-    const headerLongText = group.negate ? descriptor.negatedName : descriptor.longName;
+    const hasNot = !readOnly && 'negatedName' in descriptor && descriptor.negatedName;
```

Comment:
Nit: my early Monday morning brain impression of this was "Has not... Has not _what_?"

What do you think about `canNegate` as alternative wording. Does that convey the same meaning?

Reply:
Yeah, it reads rough to me on Monday morning.

For the sake of alternatives, can you think of any fluent names with `has` or `is` as prefix?

Reply:
I think `hasNegation` works as well as `canNegate`. Probably better than `isNegatable`.